packages/shared/src/modules/onboarding/libs/types/onboarding-question-dto.type.ts

```diff
@@ -0,0 +1,10 @@
+import { type OnboardingAnswerDto } from "./onboarding-answer-dto.type.js";
+
+type OnboardingQuestionDto = {
```

Comment:
```suggestion
type OnboardingQuestionResponseDto = {
```
If it is a response dto